transformers/events/vat_heal/converter_test.go

```diff
@@ -20,27 +20,25 @@ import (
 	"github.com/ethereum/go-ethereum/common"
 	"github.com/ethereum/go-ethereum/core/types"
 	"github.com/makerdao/vdb-mcd-transformers/test_config"
-	"github.com/makerdao/vulcanizedb/pkg/datastore/postgres"
-	. "github.com/onsi/ginkgo"
-	. "github.com/onsi/gomega"
-
 	"github.com/makerdao/vdb-mcd-transformers/transformers/events/vat_heal"
 	"github.com/makerdao/vdb-mcd-transformers/transformers/shared/constants"
 	"github.com/makerdao/vdb-mcd-transformers/transformers/test_data"
 	"github.com/makerdao/vulcanizedb/pkg/core"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
 )
 
 var _ = Describe("VatHeal converter", func() {
 
 	var (
-		converter vat_heal.Converter
-		db        *postgres.DB
+		converter = vat_heal.Converter{}
+		db        = test_config.NewTestDB(test_config.NewTestNode())
```

Comment:
In this case, I wonder if we could perhaps omit the `db` var altogether and just pass `nil` for that arg in the tests, since it's not used by the converter